tests/testthat/test-standard-entries.R

```diff
@@ -0,0 +1,153 @@
+# https://ctan.javinator9889.com/biblio/bibtex/base/btxdoc.pdf
+
+test_that("Test article-full", {
+  bib <- system.file("bib/xampl_standard.bib", package = "bibtex")
+
+  s <- expect_message(read.bib(bib))
+
+  exp_mes <- "ignoring entry 'conference-full' (line 43) because :\n\t'bibtype' has to be one of Article, Book, Booklet, InBook, InCollection, InProceedings, Manual, MastersThesis, Misc, PhdThesis, Proceedings, TechReport, Unpublished\n\n"
+
+  expect_equal(s$message, exp_mes)
+
+  out <- suppressMessages(read.bib(bib))
+
+  expect_length(out, 13)
+
+  entry <- out["article-full"]
+
+  expect_snapshot_output(toBibtex(entry))
+})
+
+test_that("Test book-full", {
+  bib <- system.file("bib/xampl_standard.bib", package = "bibtex")
+
+  out <- suppressMessages(read.bib(bib))
```

Comment:
Is there a reason we are suppressing diagnostic messages here? Should we just set `verbose = FALSE`?

Reply:
This was for not cluttering the test results with messages. I have divided the xampl in two, one with messages (tested on the first tests) and another without them.